lib/logger-helper.js

```diff
@@ -167,7 +167,7 @@ function shouldBeParsed(maskFields, excludeFields) {
 }
 
 function getResponseAudit(req, res, options) {
-    var headers = _.get(res, '_headers');
+    var headers = res && res.hasOwnProperty('_headers') && 'function' === typeof res.getHeaders ? res.getHeaders() : undefined;
```

Comment:
returning `undefined` is exactly the same behavior with how `_.get(res, '_headers')` works.

Reply:
I think this can be simplified a bit with :
```suggestion
    var headers = res && 'function' === typeof res.getHeaders ? res.getHeaders() : _.get(res, '_headers');
```
keeping backward compatibility with previous way of obtaining the headers if by any chance there would be node < 7.7.

Reply:
It's done. Thank you.